### PR TITLE
Update installation-azure-limits.adoc

### LINKED ISSUE #17762: [enterprise-4.2] Edit suggested in file installing/installing_azure/installing-azure-account.adoc
### Which section(s) is the issue in?
Azure account limits

### What needs fixing?
Default limits vary by offer category types, such as Free Trial and Pay-As-You-Go, and by series, such as Dv2, F, and G. For example, the default for Enterprise Agreement subscriptions is 350 core.
check [Subscription limits - Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#subscription-limits---azure-resource-manager)

### PR DESCRIPTION
**Pull request by hkamel**
Updating the Azure limits sections

Fixes #17762

### CODE REVIEW

File: modules/installation-azure-limits.adoc [hunk 1]
```diff
@@ -12,7 +12,9 @@ affect your ability to install {product-title} clusters.
 
 [IMPORTANT]
 ====
-You must increase quota limits for your account before you install a default
+Default limits vary by offer category types, such as Free Trial and Pay-As-You-Go, and by series, such as Dv2, F, and G. For example, the default for Enterprise Agreement subscriptions is 350 core.
+
+Check the limits based on your subscription type and if needed increase quota limits for your account before you install a default
```
Comment (kalexand-rh) [inline modules/installation-azure-limits.adoc]: s/based on/for
s/and if needed/and, if necessary,
~default~

File: modules/installation-azure-limits.adoc [hunk 2]
```diff
@@ -12,7 +12,9 @@ affect your ability to install {product-title} clusters.
 
 [IMPORTANT]
 ====
-You must increase quota limits for your account before you install a default
+Default limits vary by offer category types, such as Free Trial and Pay-As-You-Go, and by series, such as Dv2, F, and G. For example, the default for Enterprise Agreement subscriptions is 350 core.
```
Comment (kalexand-rh) [inline modules/installation-azure-limits.adoc]: core or cores?

**Review by kalexand-rh — COMMENTED** [review]
@hkamel, will you please update the PR to address a few style issues?

**Comment by kalexand-rh** [comment]
@e-tienne, will you PTAL?

File: modules/installation-azure-limits.adoc (hunk 2 above)
Comment (hkamel) [reply modules/installation-azure-limits.adoc]: cores

**Comment by kalexand-rh** [comment]
Thank you both! Merging.

**Pull request merged**